tests/sentry/replays/test_project_replay_clicks_index.py

```diff
@@ -0,0 +1,225 @@
+import datetime
+from unittest import mock
+from uuid import uuid4
+
+import pytest
+from django.urls import reverse
+
+from sentry.replays import tasks
+from sentry.replays.testutils import mock_replay, mock_replay_click
+from sentry.testutils import APITestCase, ReplaysSnubaTestCase
+from sentry.testutils.silo import region_silo_test
+from sentry.utils import kafka_config
+
+REPLAYS_FEATURES = {"organizations:session-replay": True}
+
+
+@pytest.fixture(autouse=True)
```

Comment:
is this needed here? i think it was only used for the deletion test in replay details